integration-tests/src/test/java/oracle/kubernetes/operator/ItPodsShutdown.java

```diff
@@ -261,166 +225,168 @@ private static boolean checkShutdownUpdatedProp(String podName, String... props)
-      Assertions.assertTrue(checkShutdownUpdatedProp(domainUid + "-managed-server1", "Forced"));
+      domain = createDomain(domainNSShutOpMS, shutdownPropOpt);
+      Assertions.assertNotNull(domain);
+      Assertions.assertTrue(checkShutdownUpdatedProp(domain.getDomainUid()
+          + "-admin-server", "Graceful"));
+      Assertions.assertTrue(
+          checkShutdownUpdatedProp(domain.getDomainUid()
+              + "-managed-server1", domain.getDomainNs(),"Forced", "160", "true"));
     } finally {
       LoggerHelper.getLocal().log(
-          Level.INFO, "Reverting back the domain to old crd\n kubectl apply -f {0}", originalYaml);
-      resetDomainCrd();
+          Level.INFO, " Deleting domain " + domain.getDomainUid());
+      if (domain != null) {
+        TestUtils.deleteWeblogicDomainResources(domain.getDomainUid());
+      }
     }
     LoggerHelper.getLocal().log(Level.INFO, "SUCCESS - {0}", testMethodName);
   }
 
   /**
-   * Add shutdown options at domain level and verify all pods are Terminated and recreated.
+   * Start domain with added shutdown options at the cluster level
+   * and verify values are propagated to all managed servers in the cluster.
    *
-   * @throws Exception when domain.yaml cannot be read or modified
+   * @throws Exception when domain cannot be started or failed to verify shutdown options
    */
   @Test
-  public void testAddShutdownOptionsToDomain() throws Exception {
-
+  public void testAddShutdownOptionToCluster() throws Exception {
     Assumptions.assumeTrue(FULLTEST);
     String testMethodName = new Object() {
     }.getClass().getEnclosingMethod().getName();
     logTestBegin(testMethodName);
 
-    Files.createDirectories(Paths.get(shutdownTmpDir));
-    // Modify the original domain yaml to include shutdown options in domain spec node
-    DomainCrd crd = new DomainCrd(originalYaml);
-
     Map<String, Object> shutdownProps = new HashMap();
-    shutdownProps.put("timeoutSeconds", 160);
-    crd.addShutdownOptionToDomain(shutdownProps);
+    shutdownProps.put("timeoutSeconds", 60);
+    shutdownProps.put("shutdownType", "Forced");
+    shutdownProps.put("ignoreSessions", true);
+    List<Map<String,Object>> shutdownPropsClusters = new ArrayList<>();
+    Map<String, Object> shutdownPropsMyCluster = new HashMap();
+    shutdownPropsMyCluster.put("cluster-1", shutdownProps);
+    shutdownPropsClusters.add(shutdownPropsMyCluster);
+    Map<String, Object> shutdownPropOpt = new HashMap();
+    shutdownPropOpt.put("cluster",shutdownPropsClusters);
+    Domain domain = null;
     try {
-      updateCrdYamlVerifyShutdown(crd, 0);
-      Assertions.assertTrue(checkShutdownUpdatedProp(domainUid + "-admin-server", "160"));
-      Assertions.assertTrue(checkShutdownUpdatedProp(domainUid + "-managed-server1", "160"));
+      domain = createDomain(domainNSShutOpCluster,shutdownPropOpt);
+      Assertions.assertNotNull(domain);
```

Comment:
please add messages to your assertions, so that we get a descriptive message in the log if the assertion fails. 
please consider static importing Assertions so that you can just write `assertNotNull(domain, "descriptive failure message")`